### PR TITLE
[Protocol3] Add unit tests for UserStakingPool (#266)

### PR DESCRIPTION
**Pull request by adamchen0510**

### CODE REVIEW

**Comment by dong77** [comment]
@kongliangzhong please carefully review this :)

**Comment by adamchen0510** [comment]
Add multiple users staking

File: packages/loopring_v3/test/testUserStakingPool.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        userstaking,
+        "LRCWithdrawn",
+        web3.eth.blockNumber
+      );
+      const items = eventArr.map((eventObj: any) => {
+        const withdraw = eventObj.args.amount;
+        assert.equal(withdraw, 1070, "User withdraw should equal to expected");
+      });
+    });
+  });
```
Comment (letsgoustc) [inline packages/loopring_v3/test/testUserStakingPool.ts]: It's better to add stake TRANSFER_FAILURE and withdraw INSUFFICIENT_FUND branches.

Comment (adamchen0510) [reply packages/loopring_v3/test/testUserStakingPool.ts]: Done in  statkeA test

**Pull request merged**